twitter.html

```diff
@@ -0,0 +1,349 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Twitter Clone</title>
+    <script src="https://kit.fontawesome.com/ecdbe10a83.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="twitter.css">
+    <link rel="icon" type="image/twitter-icon" href="./assets/icons/icons8-twitter-48.png">
+</head>
+<body>
+    <div class="header">
+        <div class="left-header">
+            <a href="./twitter.html" class="logo"><i class="fa-brands fa-twitter"></i></a>
+        </div>
+        <div class="main-header a">
+            <a href="#" class="home"><h3>Home</h3></a>
+            <a href="#"><i class="fa-solid fa-timeline" style="color: #00acee;"></i>
+            </a>
+        </div>
+        <div class="right-header">
+            &nbsp
+            <i class="fa-solid fa-magnifying-glass" style="color: #717274;"></i>
+            &nbsp
+            <input type="search" name="search-twitter" id="searchbar" placeholder="Search Twitter">
+        </div>
+    </div>
```

Comment:
Html is really easy to understand due to how you've set it up. Definitely helped me on how i can improve mine.